default/stringtables/en.txt

```diff
@@ -12280,19 +12289,7 @@ FU_BASIC_TANK
 Extra Fuel Tank
 
 FU_BASIC_TANK_DESC
-Increases [[metertype METER_FUEL]] capacity, allowing an additional jump.
-
-FU_DEUTERIUM_TANK
```

Comment:
Please remove this and other removed entry from other languages files.

Reply:
We still need the extra fuel tanks. 

Just deuterium and antimatter ship parts are gone. Stringtables are done.